src/Lykke.Logs/Slack/LykkeLogToSlack.cs

```diff
@@ -154,13 +178,29 @@ private string GetComponentNamePrefix()
         private string GetComponentName(string component)
         {
             if (AppEnvironment.Name == null || !AppEnvironment.Name.StartsWith(component))
-            {
                 return string.Concat(_componentNamePrefix, " : ", component);
+            return _componentNamePrefix;
+        }
+
+        private bool IsSameMessage(LogLevel level, string message)
```

Comment:
If we'll filter just by entire message entries like these:
https://lkeslacknotifications.blob.core.windows.net/slack-notifications-full-messages-2018-04/2018-04-19T15-59-56.0605780.16.txt
https://lkeslacknotifications.blob.core.windows.net/slack-notifications-full-messages-2018-04/2018-04-19T15-59-56.3016977.17.txt
will pass though this filter.
Consider to add component-wide throttling with based on messages per time interval configuration, instead of simple duplications filtering.